README.rst

```diff
@@ -3,10 +3,21 @@ k8s - Python client library for the Kubernetes API
 
 |Build Status| |Codacy Quality Badge| |Codacy Coverage Badge|
 
-
 .. |Build Status| image:: https://semaphoreci.com/api/v1/fiaas/k8s/branches/master/badge.svg
     :target: https://semaphoreci.com/fiaas/k8s
 .. |Codacy Quality Badge| image:: https://api.codacy.com/project/badge/Grade/cb51fc9f95464f22b6084379e88fad77
     :target: https://www.codacy.com/app/mortenlj/k8s?utm_source=github.com&utm_medium=referral&utm_content=fiaas/k8s&utm_campaign=badger
 .. |Codacy Coverage Badge| image:: https://api.codacy.com/project/badge/Coverage/cb51fc9f95464f22b6084379e88fad77
     :target: https://www.codacy.com/app/mortenlj/k8s?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=fiaas/k8s&amp;utm_campaign=Badge_Coverage
+
+Documentation
+    https://k8s.readthedocs.io
+Code
+    https://github.com/fiaas/k8s
+
+k8s is a python client library for Kubernetes developed as part of the FiaaS project at FINN.no AS, Norways leading classifieds site. The library has a different approach to working with the API, trying to provide an intuitive developer experience. Our approach does not allow us to use Swagger to auto-generate a library that covers the entire API, but the parts we have implemented are (in our opinion) easier to work with than the client you get when using Swagger.
```

Comment:
Norway's.

Maybe also just write FINN.no, rather than tacking on the AS?

> The library has a different approach to working with the API

Different than what? (I think it explains at the end of the paragraph, but this could be reworded for improved clarity)

Reply:
Fixed errors, reworded the paragraph a little. I'm not sure what more to put in the README. We have a tutorial and a developer guide that we point people to, what other things should we add?